test/src/test/java/org/corfudb/runtime/view/ManagementViewTest.java

```diff
@@ -1499,6 +1501,113 @@ public void verifyStateTransferNotHappenButMergeSucceeds() throws Exception {
         waitForLayoutChange(l -> l.getSegments().size() == 1, rt);
     }
 
+    /**
```

Comment:
Could we add a test for competing state transfers? that certainly they do not block each other.

Reply:
Done.